sql/core/src/test/scala/org/apache/spark/sql/StreamTest.scala

```diff
@@ -142,7 +142,10 @@ trait StreamTest extends QueryTest with Timeouts {
   case object StopStream extends StreamAction with StreamMustBeRunning
 
   /** Starts the stream, resuming if data has already been processed.  It must not be running. */
-  case object StartStream extends StreamAction
+  case class StartStream(trigger: Trigger = null, triggerClock: Clock = null) extends StreamAction
```

Comment:
You can use the same default values of `StreamExecution`. Then you don't need to handle the `null` case.